idd/Energy+.idd.in

```diff
@@ -53854,12 +53866,16 @@ Coil:Cooling:DX:VariableSpeed,
         \note curve = a + b*wb + c*wb**2 + d*odb + e*odb**2 + f*wb*odb
         \note wb = entering wet-bulb temperature (C)
         \note odb = air entering temperature seen by the condenser (C)
-   A49; \field Speed 10 Energy Input Ratio Function of Air Flow Fraction Curve Name
+   A49, \field Speed 10 Energy Input Ratio Function of Air Flow Fraction Curve Name
         \type object-list
         \object-list UnivariateFunctions
         \note quadratic curve = a + b*ffa + c*ffa**2
         \note cubic curve = a + b*ffa + c*ffa**2 + d*ffa**3
         \note ffa = Fraction of the full load Air Flow
+   A50; \field Outdoor Temperature Dependent Crankcase Heater Capacity Curve Name
+        \note A Curve:* or Table:Lookup object encoding the relationship between
+        \note the crankcase heater capacity and the outdoor air temperature. When this field is
+        \note missing or empty, constant crankcase heater capacity will be assumed.
```

Comment:
This input field should be moved up above the speed fields. It looks strange here. A transition to insert a field is not that hard. And once you do that, consider moving this field next to the Crankcase Heater Capacity input for all coil types. I won't force that, but it would then be consistent for all coil types.

Reply:
@rraustad @mjwitte I have moved the "Outdoor Temperature Dependent Crankcase Heater Capacity Curve Name" to right after the "Crankcase Heater Capacity" field and added the transition code. Would you mind taking another look? Thanks. Currently, there are two regression diffs (regression.SingleFamilyHouse_HP_Slab and regression.SingleFamilyHouse_HP_Slab_Dehumidification) with idd difference due to the moving of the curve name field.

```
--- 
+++ 
@@ -3823,6 +3823,7 @@
     ,                        !- Outdoor Dry-Bulb Temperature to Turn On Compressor {C}
     5.0,                     !- Maximum Outdoor Dry-Bulb Temperature for Defrost Operation {C}
     200.0,                   !- Crankcase Heater Capacity {W}
+    ,                        !- Outdoor Temperature Dependent Crankcase Heater Capacity Curve Name
     10.0,                    !- Maximum Outdoor Dry-Bulb Temperature for Crankcase Heater Operation {C}
     ReverseCycle,            !- Defrost Strategy
     OnDemand;                !- Defrost Control
@@ -3842,6 +3843,7 @@
     Zone Exhaust Node1_unit1,!- Evaporator Air Inlet Node Name
     Heat Pump Water Heater Fan Inlet Node_unit1,  !- Evaporator Air Outlet Node Name
     ,                        !- Crankcase Heater Capacity {W}
+    ,                        !- Outdoor Temperature Dependent Crankcase Heater Capacity Curve Name
     10,                      !- Maximum Ambient Temperature for Crankcase Heater Operation {C}
     WetBulbTemperature,      !- Evaporator Air Temperature Type for Curve Objects
     HPWH-Htg-Cap-fT,         !- Heating Capacity Function of Temperature Curve Name
```